src/test/java/com/thebois/models/world/StructureTest.java

```diff
@@ -0,0 +1,40 @@
+package com.thebois.models.world;
+
+import org.junit.jupiter.api.Test;
+
+import com.thebois.models.Position;
+import com.thebois.models.world.structures.Structure;
+import com.thebois.models.world.structures.StructureType;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StructureTest {
+
+    @Test
+    public void getPositionTest() {
+
+        // Arrange
```

Comment:
Am I the only one who doesn't like this space?

Reply:
Space as in the new line between or space between // and Arrange?

Reply:
The newline, `// Arrange` is correct.